### PR TITLE
Development

### PR DESCRIPTION
**Pull request by issa-me-sush**
- [x] minor dropdown spacing fixes

### CODE REVIEW

**Comment by lazycoder1** [comment]
Can we have the optimised/none filter enabled by default?

File: src/components/common/apiCalls/jiffyApis.tsx [hunk 1]
```diff
@@ -610,7 +610,7 @@ export const getUserOp = async (userOpHash: string, toast: any, Authorization?:
     const header = {
         'x-api-key': 'gFQghtJC6F734nPaUYK8M3ggf9TOpojkbNTH9gR5',
     } as Headers;
-
+    console.log("in api : ",userOpHash)
```
Comment (lazycoder1) [inline src/components/common/apiCalls/jiffyApis.tsx:613]: could we delete this please. We can see the results in the network tab.

File: src/components/common/apiCalls/jiffyApis.tsx [hunk 2]
```diff
@@ -625,6 +625,8 @@ export const getUserOp = async (userOpHash: string, toast: any, Authorization?:
         // if (data.userOps.length == 0) {
         //     showToast(toast, 'Error fetching data');
         // }
+        
+        console.log("fetched in api",data.userOps[0].userOpHash)
```
Comment (lazycoder1) [inline src/components/common/apiCalls/jiffyApis.tsx:629]: this too

Comment (issa-me-sush) [reply src/components/common/apiCalls/jiffyApis.tsx]: done in latest

**Pull request merged**